ql/src/java/org/apache/hadoop/hive/ql/exec/tez/DagUtils.java

```diff
@@ -708,7 +709,8 @@ Map<String, String> getContainerEnvironment(Configuration conf, boolean isMap) {
    * are set
    */
   private static String getContainerJavaOpts(Configuration conf) {
-    String javaOpts = HiveConf.getVar(conf, HiveConf.ConfVars.HIVE_TEZ_JAVA_OPTS);
+    String javaOpts = StringUtils.defaultString(HiveConf.getVar(conf, HiveConf.ConfVars.HIVE_TEZ_JAVA_OPTS));
+    javaOpts = String.join(" ", javaOpts, JavaVersionUtils.getAddOpensFlagsIfNeeded());
```

Comment:
I found this disables the warning from L728. Probably, we should update not `javaOpts` but `finalOpts`.

```
// Probablly, better to trim it because some later steps may do empty-checks
finalOpts = String.join(" ", finalOpts, JavaVersionUtils.getAddOpensFlagsIfNeeded()).trim();
LOG.debug("Tez container final opts: {}", finalOpts);
return finalOpts;
```